presto-docs/src/main/sphinx/presto_cpp/properties-session.rst

```diff
@@ -439,3 +439,11 @@ If set to ``true``, enables scaling the table scan concurrency on each worker.
 Controls the ratio of available memory that can be used for scaling up table scans.
 A higher value allows more memory to be allocated for scaling up table scans,
 while a lower value limits the amount of memory used.
+
+``native_request_data_sizes_max_wait_sec``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+* **Type:** ``integer``
+* **Default value:** ``10``
+
+Maximum wait time for exchange request in seconds.
```

Comment:
Can you please add what happens when the limit is exceeded?

Reply:
This is the max wait time for exchange long-poll protocol

Reply:
Can you expand the description (here and in the code) to make that more explicit? e.g. maximum wait time for exchange long poll requests.